### DISCUSSION TITLE
Is it possible to disable this warning?

### DISCUSSION THREAD

**Discussion by JadeTempest** [koxudaxi/datamodel-code-generator#864] [Q&A] [answered]
In our API we use the `format` field to describe formats in greater detail than the JSON schema defines, for example, `ISO 3166-1 alpha-2` for country codes and `E.164` for phone numbers. When datamodel-code-generator parses our OpenAPI document, we get this warning for them:

```
/Users/matt.holden/.cache/pre-commit/repo1eo8bl8x/py_env-python3.10/lib/python3.10/site-packages/datamodel_code_generator/parser/jsonschema.py:262: UserWarning: format of 'ISO 3166-1 alpha2' not understood for 'string' - using default
  warn(
/Users/matt.holden/.cache/pre-commit/repo1eo8bl8x/py_env-python3.10/lib/python3.10/site-packages/datamodel_code_generator/parser/jsonschema.py:262: UserWarning: format of 'ISO 4217' not understood for 'string' - using default
  warn(
```

Is there a way to disable this warning in configuration or command line? If not, would it be possible to request such a feature?

**Comment by koxudaxi** [comment, marked as answer]
@mattholden 
I'm sorry for my too-late reply.
I will add the option to disable the warn.
I have created the issue.
Thank you 
https://github.com/koxudaxi/datamodel-code-generator/issues/1025

**Comment by koxudaxi** [comment]
@mattholden   
I have released the new version as `0.16.1`
Could you please `--disable-warnings` as CLI option ?
Thank you very much!!
